views/file/addserver.html

```diff
@@ -0,0 +1,36 @@
+<div class="flex-fill w-100 d-flex flex-column">
+  <div class="modal-body container flex-fill">
+    <form>
+      <div class="form-group m-0" ng-if="!ctrl.loading">
+        <label for="serverType">Select a server type:</label>
+        <select title="ServerType" class="custom-select w-50"
```

Comment:
The `ng-change` and `ng-disabled` properties can be removed, given the error/loading state are no longer managed at this level.

Reply:
The `ng-disabled` property was originally added to prevent users from changing the server type after clicking Save. To keep that feature, I made some changes to how the loading state is set.